src/components/Header/Modal/HeaderModal.tsx

```diff
@@ -44,23 +49,33 @@ const HeaderModal = ({ content }: { content: { fields: ISection }[] }) => {
               >
                 {item.fields.children && item.fields.children.length > 0 ? (
                   <>
-                    <AccordionTrigger className="flex items-center font-inter font-semibold text-sm leading-5 px-2 py-[6px] h-[44px] hover:bg-green-neutro cursor-pointer">
+                    <AccordionTrigger
+                      className={` ${pathname.startsWith("/" + item.fields.id) ? "text-green-900" : ""} flex items-center font-inter font-semibold text-sm leading-5 px-2 py-[6px] h-[44px] hover:bg-green-neutro cursor-pointer`}
+                    >
```

Comment:
It would be better to use the cn function instead of interpolation of strings, because if the properties set overlap the cn takes care of it